lib/runner.js

```diff
@@ -114,9 +114,7 @@ app.on('ready', function() {
   parent.on('screenshot', function(path) {
     // https://gist.github.com/twolfson/0d374d9d7f26eefe7d38
     win.capturePage(function handleCapture (img) {
-      fs.writeFile(path, img, function (err) {
-        parent.emit('screenshot', arguments);
-      });
+      parent.emit('screenshot', img.toPng())
```

Comment:
ah, hmmm let's just document "only png" in the readme for now and see if it's an issue.